--- a/Makefile
+++ b/Makefile
@@ -234,7 +234,7 @@ default: all

 .PHONY: all
-all: binaries docs
+all: binaries

 .PHONY: binaries
 ifeq ($(GOOS),freebsd)
@@ -868,7 +868,7 @@
 	/usr/bin/podman info  # will catch a broken conmon

 .PHONY: install
-install: install.bin install.remote install.man install.systemd  ## Install binaries to system locations
+install: install.bin install.remote install.systemd  ## Install binaries to system locations

 .PHONY: install.remote
 install.remote:
